docs/source/api_references.rst

```diff
@@ -65,14 +65,27 @@ Callbacks API
 .. autosummary::
     :toctree: api
     :nosignatures:
+    :template: classtemplate.rst
 
-    base
-    early_stopping
-    gpu_stats_monitor
-    gradient_accumulation_scheduler
-    lr_monitor
-    model_checkpoint
-    progress
+    Callback
```

Comment:
LG! maybe order them alphabetically?

Reply:
Updated! @rohitgr7

Reply:
or let's keep Callback at first since it's the base class, then all others a-z?

Reply:
That's inconsistent.

Reply:
Should we move deprecated callbacks to the bottom? Or mark them as such on the docs page? Or will those automatically have the warning in the class docs?

Reply:
yes they will show up in docs:
<img width="857" alt="Screenshot 2022-01-06 at 11 09 16 PM" src="https://user-images.githubusercontent.com/30778939/148426478-c85d118e-321c-44b8-8ee2-d68da6351b37.png">

Reply:
if a concern is that alphabetic ordering is separating semantically related callbacks too much, you could also group them under categories. this is what we did for loops for example.

Reply:
I think the current list works. Deprecated Callbacks has the deprecation description.